openapi.yaml

```diff
@@ -313,6 +314,8 @@ x-tagGroups:
       - Market
       - Exchange
       - Exchange Ticker
+      - Index Ticker
```

Comment:
Perhaps add `BETA` on these.

Reply:
@alphafoobar I have added beta flags for index and ohlcvs respectively. If it looks good, can you please merge it to master.